packages/cli/src/util/oauth.ts

```diff
@@ -0,0 +1,307 @@
+import fetch, { type Response } from 'node-fetch';
+import { createRemoteJWKSet, jwtVerify } from 'jose';
+import ua from './ua';
+
+// https://vercel.com/.well-known/openid-configuration
+export const as: {
+  client_id: string;
+  device_authorization_endpoint: URL;
+  token_endpoint: URL;
+  revocation_endpoint: URL;
+  jwks_uri: URL;
+} = {
+  client_id: 'cl_v9RsjXlLr2yv2ZwgOSz1pe0CQvhzRn9W',
+  revocation_endpoint: new URL(
+    'https://vercel.com/api/login/oauth/token/revoke'
+  ),
+  device_authorization_endpoint: new URL(
+    'https://vercel.com/api/login/oauth/device-authorization'
+  ),
+  token_endpoint: new URL('https://vercel.com/api/login/oauth/token'),
+  jwks_uri: new URL('https://vercel.com/api/login/oauth/jwks'),
```

Comment:
```suggestion
  jwks_uri: new URL('https://vercel.com/.well-known/jwks'),
```

Reply:
Probably safer to use the discovery endpoint and not hard-code too much in CLI. Will update!